### PR TITLE
homepage modification

### PR DESCRIPTION
**Pull request by sultanassi95**
# modified homepage.
relates to: #99 

### modified:
- homepage.hbs
- homepage.css
- .travis.yml - > disabled email notifications.

### added: 
- into ./public/img/ [bg-pattern.gif, connect-to-team.png, report.png, service.png, sign-up.png]

### Modified:
- ./src/views/partials/footer.hbs
- ./public/css/homePage.css

### Added:
- ./src/public/css/footer.css

### CODE REVIEW

**Review by NouraldinS — CHANGES_REQUESTED** [review]
Great job, but un-add the ./public/css/header.css because I'm the one working on that.

**Review by NouraldinS — APPROVED** [review]
Approved.

**Comment by sultanassi95** [comment]
what the heck is wrong with travis?

"is Github down?" ! that what travis says.

File: src/tests/controllers/getTests.js [hunk 1]
```diff
@@ -14,5 +14,4 @@ test('Test Endpoint: /', (t) => {
     });
 });
 
-
 test.onFinish(() => process.exit(0));
```
Comment (mohanagy) [inline src/tests/controllers/getTests.js]: i think you must avoid using `process.exit(0);` 
you should find another way to end the test

**Review by mohanagy — CHANGES_REQUESTED** [review]

**Review by yahyaHB — APPROVED** [review]
Good job attaboy

**Review by abdhalees — APPROVED** [review]

**Pull request merged**